### DISCUSSION TITLE
Search bar enhancement

### DISCUSSION THREAD

**Discussion by JamsRepos** [ente-io/ente#1035] [Enhancements]
Quite a simple one, the ability to click an X on the right hand side within the bar to clear your current search. Often find myself holding backspace to search for another code.

**Comment by Niranjan-Dorage** [comment]
can anyone describe this issue please ?  what exactly need to be done ?
 @vishnukvmd

**Comment by JamsRepos** [comment]
> can anyone describe this issue please ? what exactly need to be done ? @vishnukvmd

Essentially on desktop, where you can type to search for a certain application, having the ability to clear the field so you can type your next application.

Often you have CTRL + A then delete to clear the field, whereas it would be better if you could just click the inside the box on the right to clear and select the field. 

<img width="347" alt="image" src="https://github.com/ente-io/auth/assets/1347620/42f653a4-7a23-4883-9db1-c40a5f022f55">

Similar to how we have the eye icon on the password field, this would be an "X" to clear the data and select the field.

**Comment by Niranjan-Dorage** [comment]
we already have X to unfocus the searchbar , and we have to make button for selecting all text or removing all text ? @JamsRepos

**Comment by JamsRepos** [comment]
> we already have X to unfocus the searchbar , and we have to make button for selecting all text or removing all text ? @JamsRepos

Correct - on the mobile app. Not on the web app which this issue is for. 

<img width="406" alt="image" src="https://github.com/ente-io/auth/assets/1347620/9e20a864-bfcf-48d2-8c71-5addcd01ccfc">

**Comment by Hudzen10** [comment]
Also, keeping the search bar visible in the Web interface when scrolling down the list would be handy as well.
